Specs/Number/Japanese/NumberRangeModel.json

```diff
@@ -2095,36 +1997,32 @@
   },
   {
     "Input": "+1-222-2222-2222は電話番号です。",
-    "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "Results": []
   },
   {
     "Input": "+1-222-2222/2222は電話番号です。 ",
-    "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "Results": []
   },
   {
     "Input": "4分の1は分数です。",
-    "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "Results": []
   },
   {
     "Input": "1995-01",
-    "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "Results": []
   },
   {
     "Input": "1995年1月",
-    "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "Results": []
   },
   {
     "Input": "5k-20k",
+    "Comment": "PendingValidation",
```

Comment:
This came from a chat with a Japanese person about informal usage of ranges in business emails. I think we can keep it.

Reply:
Sure @tellarin .